pkgs/by-name/ei/eiffel-studio-bin/package.nix

```
@@ -0,0 +1,70 @@
+{
+  autoPatchelfHook,
+  gcc,
+  gnumake,
+  gtk2-x11,
+  lib,
+  makeWrapper,
+  stdenv,
+  xorg,
+  build ? "103187",
+  libraries ? [ ],
+  platform ? "linux-x86-64",
+  version ? "19.05",
+}:
+
+stdenv.mkDerivation {
+  pname = "eiffelstudio-bin";
+  version = version;
+  src = fetchTarball {
+    url = "mirror://sourceforge/eiffelstudio/EiffelStudio%20${version}/Build_${build}/Eiffel_${version}_gpl_${build}-${platform}.tar.bz2";
+    sha256 = "sha256:1m96qg581m1hfjaycc0i0v96vlhs7y0nl9slxk0qb24x3ki2xpjf";
```

```suggestion
    hash = "";
```
also generate hash from a build error (`nix-build -A eiffel-studio-bin.src`)